src/dbwrapper.h

```diff
@@ -17,6 +17,9 @@
 #include <leveldb/db.h>
 #include <leveldb/write_batch.h>
 
+static const size_t DBWRAPPER_PREALLOC_KEY_SIZE = 64;
+static const size_t DBWRAPPER_PREALLOC_VALUE_SIZE = 1024;
```

Comment:
I see no reason not to go higher here? Keys should ~always be 35 bytes (ish, but 64 is fine), but values could be higher...but might as well also prealloc more than 1K for values since they're occasionally higher and the memory usage isnt an issue. Did you benchmark this?